GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -256,6 +256,18 @@ gdjs.RuntimeGamePixiRenderer.prototype.setFullScreen = function(enable) {
   }
 };
 
+/**
+ * Checks if the game is in full screen.
+ */
+gdjs.RuntimeGamePixiRenderer.prototype.isFullScreen = function() {
+  var electron = this.getElectron();
+  if (electron) {
+    return electron.remote.getCurrentWindow().isFullScreen();
+  }
+  // Height check is used to detect F11 triggered full screen.
```

Comment:
Nit picking: "[...] user triggered full screen (for example F11 shortcut)"

F11 could change and is not the only way to trigger it (menu bar for example)